src/ripple/basics/impl/contract.cpp

```diff
@@ -39,6 +41,18 @@ accessViolation() noexcept
 
 } // detail
 
+void
+LogThrow (std::string title)
+{
+    if (auto j = debugJournal().debug())
+    {
+        j << title;
+        for (auto const& frame : beast::getStackBacktrace ())
+            j << "    " << frame;
+        j << "----";
+    }
+}
```

Comment:
Thanks.  I think I'm catching on.  The problem is that once the `catch` occurs the stack below is lost.  We can't see where the `throw` came from.  All we can log is that we are where we are and we have this $%^& exception in out hands.